src/helpers/select-helpers.ts

```diff
@@ -126,7 +123,20 @@ function generateModelArgsInputObjectTypes(models: DMMF.Model[]) {
     };
     fields.push(selectField);
 
-    /** @todo add include field in a future pull request */
+    const includeField: DMMF.SchemaArg = {
```

Comment:
This function `generateModelArgsInputObjectTypes` inside `select-helpers` is no longer being used here after it was moved outside. So let's remove it please.

Reply:
Done